### ISSUE TITLE
Strange check in `recipe._process_zope_vhm_map`

### ISSUE THREAD

**Issue by jean** [collective/plone.recipe.varnish#19]
My buildout fails with: 
_varnish-configuration: When using VHM a hostname must be given for each backend_

The code is this:

``` python
        # consistency checks
        if result:
            lengths = set([len(x) for x in backends])
            if lengths != set([3]):
                self._log_and_raise(
                    'When using VHM a hostname must be given for each backend'
                )
```

My backends look like this:

``` python
(Pdb) pprint(backends[0])
{'between_bytes_timeout': '60s',
 'connect_timeout': '0.4s',
 'first_byte_timeout': '300s',
 'host': '127.0.0.1',
 'name': 'backend_000',
 'port': '10388',
 'url': 'abnj.iwlearn.org'}
```

How is a check for `set(len[3])` going to tell that the hostname specifically is missing? 
Where are the other keys coming from?

**Comment by cekk** [comment]
@mauritsvanrees what about a new release with this fix?

**Comment by mauritsvanrees** [comment]
Yes, let's do a release soon.  I merged a few more PRs today, and created PR #96 to update to Varnish 6.0.11.  Let's get that in first.
